edifact-edi/2.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,107 @@
+|Messages |A hierarchy of messages that are either read by the module during a read operation or sent by the module during a write operation. The value of this top-level key is a map with standard versions as keys, in the form `D96A`. For example, if you are using version `D96.A ORDERS` and `CUSRES` messages, the Messages contain a map with one key, `D96A`. The value of this key is another map, one with two keys, `ORDERS` and `CUSRES`. Each of these contains a list of individual ORDERS and CUSRES messages that are processed.
+|===
+
+Individual messages have their own maps, with the following keys:
+
+[%header%autowidth.spread]
+|===
+|Key |Description
+|Detail |Map of segments or loops from the detail section of the message. Values are maps for segments or loops that occur once. Lists of maps are for values that occur more than once.
+|Errors (read only) |A list of errors associated with the message.
+|Heading |A map of segments or loops from the heading section of the message. Values are maps for segments or loops that occur once. Lists of maps exist for values that occur more than once.
+|Id |The Message ID must match the key of the containing message list.
+|Interchange |A map of UNB interchange header segment values. For a read message, the map is the actual data from the enclosing interchange. It is a single map linked from all messages in the interchange. For a write message, the map value creates the enclosing interchange. For example, gathering messages with the same interchange values into a single interchange, regardless of whether the actual maps are the same. If there are no values for this map, then the default is the value(s) taken from the Interchange map at the message level.
+|MessageHeader |Map of UNH message header segment values. This provides the actual header data for a Read operation, and enables you to provide overrides for for Write operation configuration settings.
+|Name |Message name
+|Summary |Map of segments or loops from the summary section of the message. Values are maps for segments or loops that occur once. Lists of maps are for values that occur more than once.
+|===
+
+Generated CONTRL functional acknowledgment messages differ from Received messages in their handling of interchange information:
+
+[%header%autowidth.spread]
+|===
+|Key name |Description
+|Interchange |For functional acknowledgments generated by Receive processing, this map is a copy of the data for the containing interchange with Sender and Receiver identification components (UNB2.1/UNB2.2 and UNB2.1/UNB3.2). When writing a message, these values are used for creating the enclosing interchange. For example, gathering messages with the same interchange values into a single interchange, regardless of whether the actual maps are the same. If there are no values for this map, then the default is the value(s) taken from the Interchange map at the message level.
```

Comment:
unclear reference: "When writing a message, these values are used...." What is writing the message?

Reply:
Fragment:
For example, gathering messages with the same interchange values into a single interchange, regardless of whether the actual maps are the same.

Reply:
ditto re. value or values rather than value(s); GLOBAL